software/oada_upload/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "target": "es6",
+    "lib": ["es2019"],
+    "esModuleInterop": true,
```

Comment:
Do things break if you delete this line? Seems better to not have it on if possible.

Reply:
Does not break things, did previously. Removed

Reply:
It appears that the moment package breaks when this is removed. I added this back in